docs/contributing.rst

```diff
@@ -1,2 +1,37 @@
 Contributing
 ============
+
+We want you to contribute to Hydroshare development, and we want you to be
+as successful as possible in doing so. This document's aim is to provide you
+with everything you need to succeed in your contributions.
+
+Getting Started
+---------------
+
+#) Fork the repo under your own name
+
+#) Set up your Hydroshare development environment by following the `installation directions`_
+
+#) Make the required edits to the code
+
+#) Push to your fork and `submit a pull request`_
+
+.. _installation directions: https://github.com/hydroshare/hydroshare#simplified-installation-instructions
+.. _submit a pull request: https://github.com/hydroshare/hydroshare/compare/
+
+After You've Submitted a Pull Request
+-------------------------------------
+
+The Hydroshare team varies in size and responsiveness over time. However, we try
```

Comment:
I think that this could be simplified to simply: "We do our best to respond to pull requests within three business days"